src/LaravelRequestDocs.php

```diff
@@ -182,6 +189,13 @@ public function getControllersInfo(array $onlyMethods): Collection
                 $controllerName     = (new ReflectionClass($controllerFullPath))->getShortName();
             }
 
+            $pathParameters = [];
+            $pp             = $this->routePath->getPathParameters($route);
+            // same format as rules
```

Comment:
failing tests are caused by this new change
https://github.com/rakutentech/laravel-request-docs/pull/204#discussion_r1147161924

Reply:
fixed